### PR TITLE
Fix the documentation for functions in Base

### PR DESCRIPTION
**Pull request by alco**

### CODE REVIEW

File: lib/elixir/lib/base.ex [hunk 1]
```diff
@@ -150,8 +150,10 @@ defmodule Base do
   @doc """
   Encodes a binary string into a base 16 encoded string.
 
-  Accepts an atom `:upper` (default) for encoding to upper case characters or
-  `:lower` for lower case characters.
+  ## Options
```
Comment (lexmag) [inline lib/elixir/lib/base.ex]: Maybe we should reuse the format from the https://github.com/elixir-lang/elixir/blob/master/lib/elixir/lib/file.ex#L257-L266?
To me it seems easier to read.

**Comment by josevalim** [comment]
@alco ping. :)

**Comment by alco** [comment]
Updated.

**Comment by whatyouhide** [comment]
Looks better to me :)

**Comment by josevalim** [comment]
@alco :shipit: :)

**Pull request merged**